quirks/password-rules.json

```diff
@@ -92,6 +92,9 @@
     "comcastpaymentcenter.com": {
         "password-rules": "minlength: 8; maxlength: 20; required: lower, upper; required: digit; max-consecutive: 2;"
     },
+    "comdirect.de": {
+        "password-rules": "minlength: 6; maxlength: 6; required: digit; digit; digit; digit; digit; digit; "
```

Comment:
This doesn’t look right to me.

Reply:
I think this rule would accomplish the same thing:

```
minlength: 6; maxlength: 6; required: digit;
```

Reply:
I think allowed: digit would do the work.

Reply:
True!